src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/MySqlEngine.java

```diff
@@ -733,6 +734,53 @@ protected void dropReferringFks(DbEntity entity) throws DatabaseEngineException
         }
     }
 
+    @Override
+    public synchronized Map<String, DbEntityType> getEntities(final String schemaPattern) throws DatabaseEngineException {
+        final Map<String, DbEntityType> entities = new LinkedHashMap<>();
+        ResultSet rs = null;
+
+        try {
+            getConnection();
+
+            // get the entities
+            rs = conn.getMetaData().getTables(schemaPattern, null, "%", null);
```

Comment:
Meh, just because of this difference (when compared to the same method of the abstract class), but yeah..

Could you please put the finals where necessary in this method?